nixos/modules/services/monitoring/graphite.nix

```diff
@@ -413,6 +418,11 @@ in {
       cfg.carbon.enableCache || cfg.carbon.enableAggregator || cfg.carbon.enableRelay ||
       cfg.web.enable || cfg.seyren.enable
      ) {
+      systemd.slices.system-graphite = {
+        description = "Common slice for all graphite services";
```

Comment:
Ref. https://www.freedesktop.org/software/systemd/man/latest/systemd.unit.html#Description=, I think this is better named something like "Graphite Graphing System Slice".